packages/core-data/src/fetch/__experimental-fetch-link-suggestions.js

```diff
@@ -88,23 +88,23 @@ const fetchLinkSuggestions = async (
 		type = undefined,
 		subtype = undefined,
 		page = undefined,
-		perPage = isInitialSuggestions ? 3 : 20,
```

Comment:
Why did we change this?

Reply:
Why would initial suggestions be three?

Reply:
We should let this be the same and control it at every instance instead.